src/App.js

```diff
@@ -1,25 +1,32 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
-
+import React, {useState,useEffect} from 'react';
+import  'bootstrap/dist/css/bootstrap.min.css'
+import 'bootstrap/dist/js/bootstrap'
+import Routing from './Routes'
+import Request from './request'
+import './App.css'
 function App() {
+  
+  const [information, setInformation]=useState([])
+   
+  useEffect(()=>{
+  Request.get().then(data =>{
+     setInformation( data.data.record)
+  })  
+  },[])
+ 
+
+  
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
+    <Routing />
+    {
+       information.map(element=>(
+         <>
```

Comment:
we can do away with the fragments here. Also you have to pass the a unique key to every child in this list.
so you could do something like this:
```
information.map((element,key)=>(
<h1 key={key}>{element.tagline}</h1>
))
```